### PR TITLE
feature: Add generation of a "Syndesis Developer Handbook"

### PR DESCRIPTION
**Pull request by rhuss**
this is still WIP

### CODE REVIEW

**Comment by rhuss** [comment]
No wip anymore. Could please someone approve it so that I can check whether the documentation is really built ? (happens only on `master`)

File: Jenkinsfile [hunk 1]
```diff
@@ -12,7 +12,7 @@ pipeline {
     stages {
         stage('prepare') {
             steps {
-             withCredentials([string(credentialsId: "api.rh-idev.openshift.com", variable: 'TOKEN')]) {
+             withCredentials([string(credentialsId: "${env.IGNITE_ACCOUNT_CREDENTIAL}", variable: 'TOKEN')]) {
```
Comment (dsimansk) [inline Jenkinsfile]: Is this change intentional? I guess `IGNITE_ACCOUNT_CREDENTIAL` and `--server=https://api.rh-idev.openshift.com` won't work very well.

Comment (rhuss) [reply Jenkinsfile]: Yeah, not really sure why this slipped in. But this was a fix from @iocanel upstream so I wonder why it disappears here as a diff.
